server/conf/helper/feature-flags.conf

```diff
@@ -30,3 +30,7 @@ intake_form_enabled = ${?INTAKE_FORM_ENABLED}
 # api generated docs tab
 api_generated_docs_enabled = false
 api_generated_docs_enabled = ${?API_GENERATED_DOCS_ENABLED}
+
+# completed application exportable
+application_exportable = true
```

Comment:
@zarifzm  -  the feature flags are turned off by default.  We should turn it on env. basis. Example - first turn it on in staging env, test it out, then turn it on in prod.